libraries/AP_Mission/AP_Mission.cpp

```diff
@@ -771,9 +771,10 @@ MAV_MISSION_RESULT AP_Mission::mavlink_int_to_mission_cmd(const mavlink_mission_
         break;
 
     case MAV_CMD_NAV_LOITER_TO_ALT:                     // MAV ID: 31
-        cmd.p1 = fabsf(packet.param2);                  // param2 is radius in meters
+        cmd.content.loiter_to_alt.radius = fabsf(packet.param2);    // radius in meters
         cmd.content.location.loiter_ccw = (packet.param2 < 0);
```

Comment:
You can't set two different structs inside an union, one will overwrite the other.

Reply:
Yup. I know. Totally forgot. My bad.